core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedClusterMessage.scala

```diff
@@ -132,4 +132,6 @@ private[spark] object CoarseGrainedClusterMessages {
   // Used internally by executors to shut themselves down.
   case object Shutdown extends CoarseGrainedClusterMessage
 
+  // The message to check whether the executor is alive or not.
```

Comment:
might be nice just to clarify saying check if scheduler thinks executor is alive or not